docs/reference/commandline/load.md

```diff
@@ -23,7 +23,6 @@ Load an image from a tar archive or STDIN
 Options:
       --help           Print usage
   -i, --input string   Read from tar archive file, instead of STDIN.
-                       The tarball may be compressed with gzip, bzip, or xz
```

Comment:
Adding it to ` load.go` is currently problematic, because we don't have a good solution (yet) for wrapping. @ijc25 is looking into improvements in that area, but for now I think we should just keep it as-is (and add the extra information once we have a good solution)